samples/ios/Continote/Continote/MyNotesViewController.swift

```diff
@@ -92,59 +94,55 @@ class MyNotesViewController: BaseViewController {
         return cell
       })
 
-    myNotesTableViewDataSource?.bind(to: myNotesTableView)
+    dataSource?.bind(to: tableView)
 
     // Finally, show this screen's UI since everything is ready.
-    myNotesTableView.isHidden = false
-    writeNewNoteButton.isHidden = false
+    tableView.isHidden = false
+    newNoteButton.isHidden = false
   }
 
   override func handleUserSignedOut() {
     super.handleUserSignedOut()
 
     // Hide this screen's UI since the user must be signed in to see any Notes.
-    myNotesTableView.isHidden = true
-    writeNewNoteButton.isHidden = true
+    tableView.isHidden = true
+    newNoteButton.isHidden = true
 
     // The user must be signed in to view this screen, so navigate away from it if the user is
     // signed out.
     navigationController?.popViewController(animated: true)
   }
 
   /**
-   Handles when the user taps the "write a new note" button.
+   Handles when the user taps the newNoteButton.
 
    Adds a new Note to the Firebase Realtime Database for the current user,
    then opens that Note to allow the user to begin writing.
 
-   - Parameter sender: The object that called this action. This should only be the
-   writeNewNoteButton.
+   - Parameter sender: The object that called this action. This should only be the newNoteButton.
    */
-  @IBAction func writeNewNoteButtonAction(_ sender: Any) {
-    guard let notesFirebaseDatabaseRef = notesFirebaseDatabaseRef else {
+  @IBAction func newNoteButtonAction(_ sender: Any) {
+    guard let notesRef = notesRef else {
```

Comment:
Nit on style: guard statements that just return tend to be nicer on one line, where the comment can go above. In this case though I think it's pretty clear since notesRef is optional, it's a totally sane thing to guard unwrap it so you can probably remove the comment.